src/manifest.json

```diff
@@ -1,4 +1,10 @@
 ﻿{
+    "applications": {
+        "gecko": {
+            "id": "addon@darkreader.com",
+            "strict_min_version": "42.0"
```

Comment:
Did you try it on FF42? Currently I have FireFox 54 installed, let's stick to version that we can check.